bigquery/api/Snippets/LoadFromFile.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright(c) 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// [START bigquery_load_from_file]
+
+using Google.Cloud.BigQuery.V2;
+using System;
+using System.IO;
+
+public class BigQueryLoadFromFile
+{
+    public void LoadFromFile(
+        string projectId = "your-project-id",
+        string datasetId = "your_dataset_id",
+        string tableId = "your_table_id",
+        string filePath = "path/to/file.csv"
+    )
+    {
+        BigQueryClient client = BigQueryClient.Create(projectId);
+        // Create job configuration
+        var uploadCsvOptions = new UploadCsvOptions()
+        {
+            SkipLeadingRows = 1,  // Skips the file headers
+            Autodetect = true
+        };
+        using (FileStream stream = File.Open(filePath, FileMode.Open))
+        {
+            // Create and run job
+            BigQueryJob job = client.UploadCsv(
```

Comment:
So there's a special "CSV" method in the C# client? This is problematic since we only have one sample for loading local files. 

Does that mean there is a corresponding method for the other file formats? Is there a more generic method we could use?

If not, let's add a comment mentioning some of the other upload options classes & methods.

Reply:
It looks like there are methods for Avro, JSON, CSV, Parquet, and Orc starting with [UploadAvro](https://googleapis.github.io/google-cloud-dotnet/docs/Google.Cloud.BigQuery.V2/api/Google.Cloud.BigQuery.V2.BigQueryClient.html#Google_Cloud_BigQuery_V2_BigQueryClient_UploadAvro_Google_Apis_Bigquery_v2_Data_TableReference_Google_Apis_Bigquery_v2_Data_TableSchema_System_IO_Stream_Google_Cloud_BigQuery_V2_UploadAvroOptions_) until the bottom of that page in the reference docs. The generic [CreateLoadJob](https://googleapis.github.io/google-cloud-dotnet/docs/Google.Cloud.BigQuery.V2/api/Google.Cloud.BigQuery.V2.BigQueryClient.html#Google_Cloud_BigQuery_V2_BigQueryClient_CreateLoadJob_System_Collections_Generic_IEnumerable_System_String__Google_Apis_Bigquery_v2_Data_TableReference_Google_Apis_Bigquery_v2_Data_TableSchema_Google_Cloud_BigQuery_V2_CreateLoadJobOptions_) method seems to be just for files stored in GCS. I'll add a comment saying there are other formats.